test/blocks/global-footer/mocks/placeholders.js

```diff
@@ -0,0 +1,106 @@
+const placeholders = {
```

Comment:
Do we actually need this redefined, since it should already be available in https://github.com/adobecom/milo/blob/main/test/blocks/global-navigation/mocks/placeholders.js? Could we not re-use that one?